code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1,6 +1,7 @@
 /datum/reagent/blood
 	data = list("viruses"=null,"blood_DNA"=null,"blood_type"=null,"resistances"=null,"trace_chem"=null,"mind"=null,"ckey"=null,"gender"=null,"real_name"=null,"cloneable"=null,"factions"=null,"quirks"=null)
 	name = "Blood"
+	description = "A suspension of organic cells necessary for transport of oxygen. Keep inside at all times."
```

Comment:
description = "A suspension of organic cells necessary for the transport of oxygen. Keep inside at all times." would probably sound better, because of implicit grammar.

Reply:
```suggestion
	description = "A suspension of organic cells necessary for the transport of oxygen. Keep inside at all times."
```